.gitignore

```diff
@@ -1 +1,48 @@
 bower_components/
+accessibility-developer-tools/
```

Comment:
Oh, this shouldn't be needed. I'm not sure how your project is set up, but an element doesn't live in the same directory as the other elements unless you specifically made it so -- which means this kind of .gitignore doesn't live here :(